addons/udes_sale_stock/models/sale_order_line.py

```diff
@@ -134,3 +135,35 @@ def _set_ui_is_cancelled(self):
             )
 
         lines_to_cancel.action_cancel()
+
+    def write(self, values):
+        """ If quantity is reduced, stock moves are not updated for them (but warning
+        issued). Update linked stock moves if those are still editable.
+        """
+        lines_qty_reduced  = self.env["sale.order.line"]
+        moves_editable = self.env["stock.move"]
```

Comment:
Definition of class at very beginning, due to our standards of defining classes

```suggestion
        StockMove = self.env["stock.move"]
        moves_editable = Moves.browse()
```